database_schema.md

```diff
@@ -83,6 +84,33 @@ For example: a person has two terms as city council member for D4 then a term as
 Roles can also be tied to committee chairs.
 For example: a council member spends a term on the transportation committee and then spends a term on the finance
 committee.
+```
+role_id: {
+    person_id: str
+    person_name: str
+    title: str
+    body_id: str
+    body_name: str
```

Comment:
In this example, a `role` can be "City Council Member". What would the body name for that be?

Reply:
For "City Council Member" role, `title` would be 'Council Member' and `body_name` would be 'City Council'.
For "City Council President" role, `title` would be 'Council President' and `body_name` would be 'City Council'

Reply:
Sounds good. Makes sense to me.